api/tests/unit/features/test_unit_features_views.py

```diff
@@ -78,11 +82,123 @@ def test_list_feature_states_nested_environment_view_set(
     Feature.objects.create(name="another_feature", project=project)
 
     # When
-    with django_assert_num_queries(6):
+    with django_assert_num_queries(5):
```

Comment:
How come this changed?

Reply:
Ah yes, good catch, I actually thought that I updated the permission. It's because we are doing the permission check in the permission class now instead of views

Reply:
Hmm... sorry, I'm a little confused. It changed from 6 -> 5 but now it's gone from 5 -> 7 after your latest commit? Is there a way to avoid adding the extra query?

Reply:
Well, no, not without some refactor, it went from 6 to 7 because I moved the permission check from get_queryset to permissions class